sky/engine/core/events/VelocityTracker.cpp

```diff
@@ -22,11 +22,10 @@ class VelocityTrackerStrategy {
   virtual ~VelocityTrackerStrategy() {}
 
   virtual void Clear() = 0;
-  virtual void ClearPointers(BitSet32 id_bits) = 0;
   virtual void AddMovement(const base::TimeTicks& event_time,
-                           BitSet32 id_bits,
-                           const PointerXY* positions) = 0;
-  virtual bool GetEstimator(uint32_t id, Estimator* out_estimator) const = 0;
+                           int32_t id,
+                           const PointerXY &position) = 0;
+  virtual bool GetEstimator(int32_t id, Estimator* out_estimator) const = 0;
```

Comment:
Isn't there code that determines when to reset state depending on whether the pointer ID has changed since last call?